src/scenes/UserGalleryPage/UserGalleryCollection.tsx

```diff
@@ -85,24 +83,24 @@ const StyledCollectorsNote = styled(BodyRegular)`
   white-space: pre-line;
 `;
 
-const StyledCollectionNfts = styled.div`
+const StyledCollectionNfts = styled.div<{ columns: number }>`
   margin: 10px 0;
   display: flex;
   flex-wrap: wrap;
   align-items: center;
+  justify-content: ${({ columns }) => columns === 1 ? 'center' : 'initial'};
 
   // Can't use these for now due to lack of Safari support
-  // column-gap: ${GAP_PX}px;
-  // row-gap: ${GAP_PX}px;
+  // column-gap: px;
+  // row-gap: px;
```

Comment:
yea exactly, I removed the `GAP_PX` cus the const doesn't exist anymore, and because these lines are commented I didn't bother replacing the placeholder since it might change by the time we bring these back